module/icp/io/sha2_mod.c

```diff
@@ -387,11 +387,11 @@ sha2_digest_final_uio(SHA2_CTX *sha2_ctx, crypto_data_t *digest,
 			 */
 			SHA2Final(digest_scratch, sha2_ctx);
 
-			bcopy(digest_scratch, (uchar_t *)uio_iovbase(digest->
-			    cd_uio, vec_idx) + offset,
-			    digest_len);
+			bcopy(digest_scratch,
+			    (uchar_t *)zfs_uio_iovbase(digest->cd_uio, vec_idx)
+			    + offset, digest_len);
```

Comment:
I did this originally because the + made the line longer than 80 characters. I updated how this is split between lines. Double check it though with the new way I split this up between lines.